.ci/ci_cache_nemu.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# The main purpose of this script is to
+# optimise the nemu build from source and
+# use a cache as a prebuilt
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Source to trap error line number
+source "${script_dir}/../lib/common.bash"
+
+WORKSPACE=${WORKSPACE:-$(pwd)}
+nemu_dir="/usr/local/bin/"
+
+cache_built_nemu() {
```

Comment:
Please put a comment header on this function explaining:

- The arguments.
- The way the function works. Specifically detail what files are created, why and where.